problems/loops.js

```diff
@@ -212,7 +348,15 @@ function targetCount() {}
  * @param {number|string} target
  * @returns {number} first index found.
  */
-function firstIndexFound() {}
+const firstIndexFound = (elements, target) => {
+  for (let i = 0; i <= elements.length - 1; i++) {
+    if (i === target) {
+      return elements[i];
+    } else {
+      return -1;
```

Comment:
We only want to return -1 if we made it all the way through the loop and didn't find it.